PMM-13296 - Document additional Percona Alert rule templates (#1288)
percona/pmm-doc: docs/get-started/alert-templates.md

# PMM alert templates

Alert templates provide a set of common events and expressions for alerting, serving as a foundation for creating alert rules.

Percona Monitoring and Management (PMM) offers three categories of alert templates to enhance database performance monitoring:

1. **Built-in templates**: templates that are available out-of-the-box with the PMM installation and are available to all PMM users.

> 2. **Percona Platform templates**: additional templates available exclusively to Percona customers who [connect PMM to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.

Reviewer: ```suggestion
2. **Percona Platform templates**: additional templates dynamically delivered to PMM if the instance [connected to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.
```

It Percona Platfrom - it can be  "all user " templates, not customers only.

    When connected to the Platform, PMM automatically downloads these templates if the **Telemetry** option is enabled under **Configuration > Settings > Advanced Settings**.
3. **Custom templates**: user-created templates for specific needs not met by built-in or Percona Platform templates. These allow you to tailor alerts to your unique environment and requirements.
   For details on creating custom templates, see [Percona Alerting](../get-started/alerting.md#configure-alert-templates).

## Accessing alert templates

To check the alert templates for your PMM instance, go to PMM > **Alerting > Alert Rule Templates** tab.

## Available alert template

The table below lists of alert templates available in Percona Monitoring and Management (PMM). 

> This list includes both built-in templates, accessible to all PMM users, and customer-only templates, which are available to all Percona customers who have [connected PMM to Percona Platform](../how-to/integrate-platform.md) using a Percona Account, regardless of their specific tier.

Reviewer: ```suggestion
This list includes both built-in templates, accessible to all PMM users, and customer-only templates.
To have a Customer's list of Templates you need to be a percona customer and   [connect PMM to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.
```

> This list includes both built-in templates, accessible to all PMM users, and customer-only templates, which are available to all users who have [connected PMM to Percona Platform](../how-to/integrate-platform.md) using a Percona Account.

Reviewer: this sounds like a duplication of previous section
